coresdk/src/test/unit_tests/unit_test_bitmap.cpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * Bitmap Unit Tests
+ */
+
+#include "catch.hpp"
+
+#include "types.h"
+#include "graphics.h"
+#include "resources.h"
+
+using namespace splashkit_lib;
+
+TEST_CASE("bitmaps can be created and freed", "[bitmap]")
+{
+    SECTION("can detect non-existent bitmap")
+    {
+        REQUIRE(has_bitmap("non_existent") == false);
+        bitmap no_bmp = load_bitmap("non_existent", "non_existent.jpg");
+        REQUIRE(no_bmp == nullptr);
+        REQUIRE(has_bitmap("non_existent") == false);
+    }
+    SECTION("can load and free bitmap")
+    {
+        bitmap bmp;
+        SECTION("can load bitmap")
+        {
+            REQUIRE(has_bitmap("cottage") == false);
+            bmp = load_bitmap("cottage", "cottage_door_and_window_199195.jpg");
```

Comment:
Are we able to replace the filename parameter with a string variable declared just outside the section declaration?